addons/dockable_container/dockable_container.gd

```diff
@@ -161,6 +164,61 @@ func _drop_data(_position: Vector2, data) -> void:
 	queue_sort()
 
 
+func _add_floating_options(tab_container: DockablePanel) -> void:
+	var options := PopupMenu.new()
+	options.add_item("Make Floating")
+	options.id_pressed.connect(_toggle_floating.bind(tab_container))
+	options.size.y = 0
+	_windows_container.add_child(options)
+	tab_container.set_popup(options)
```

Comment:
This method could be implemented in `DockablePanel` directly, what do you think? `DockablePanel` would then have a `toggle_floating_window` signal or something like this, that already passes all required values like current tab index, current tab name, leaf control, etc... and `DockableContainer` simply connects it to `_toggle_floating`.

By the way, I think the popup doesn't need to be a child of `_windows_container` or anything else, setting a popup to a TabContainer ties the popup to it and you don't need to worry about it ever again: it will be managed by the tab container. Doesn't it? 🤔